docs/source/PerformanceTesting.rst

```diff
@@ -14,10 +14,10 @@ network
 block
 -----
 
-- `iozone (linux) <https://github.com/autotest/autotest/tree/master/client/tests/iozone/>`_
-- `iozone (windows) <https://github.com/autotest/autotest/tree/master/client/virt/tests/iozone_windows.py>`_ (iozone has its own result analysis module)
+- `iozone (linux) <https://github.com/autotest/virt-test/virttest/iozone>`_
+- iozone (windows) (iozone has its own result analysis module)
 - iometer (windows) (not push upstream)
-- `ffsb (linux) <https://github.com/autotest/autotest/tree/master/client/tests/ffsb/>`_
+- `ffsb (linux) <https://github.com/autotest/virt-test/virttest/ffsb>`_
```

Comment:
Well, one could argue that the links referring virt-test are out of date, as that repo is frozen for a while now